bootstrap/bootstrap.go

```diff
@@ -59,12 +58,43 @@ const (
 	apiTypeSignin  = "signin"
 )
 
-// bootstrap is a data structure to hold configuration required to start
+// Defaults.
+const (
+	DefaultIdentifierClientPath = "./identifier-webapp"
+	DefaultSigningKeyID         = "default"
+	DefaultSigningKeyBits       = 2048
```

Comment:
Do those consts need to be exported? It would be better to make private to avoid to suggest that these values have any meaning outside of konnect bootstrap.

Reply:
Problem is that these defaults are used by cmd package and in bootstrap, that is why I temporarily duplicated them. Any suggestions where to put them?

Reply:
Hmhh I might create env-vars for these defaults and use envOrDefault, this would then solve the problem of https://github.com/Kopano-dev/konnect/pull/19#discussion_r372814177

Reply:
I would prefer to have those private since these define internal behavior of the bootstrap package. If the cmd package needs to have its own defaults, i guess it can define it too. I would be ok with this small duplication.

Reply:
Un-exported defaults for now